server.go

```diff
@@ -176,38 +189,36 @@ func getItems(c *gin.Context) {
 	c.IndentedJSON(http.StatusOK, collection)
 }
 
-/*func getItems(c *gin.Context) {
-	var collection Collection
-	var str = readFile("list.json")
-	json.Unmarshal([]byte(str), &collection)
-	filter, status := c.GetQuery("rawFilter")
-	if !status {
-		c.IndentedJSON(http.StatusOK, collection)
+//Handler for /removeItem endpoint with DELETE method
+func removeItem(c *gin.Context) {
+	//Make initial connection with database
+	sqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
```

Comment:
It seems like this initial step of creating the database connection is repeated on every call (PATCH vs DELETE vs PUT vs GET). Perhaps its worth making this a separate function that each method calls so as to reduce code duplication?